packages/framework/attributor/src/encoders.ts

```diff
@@ -90,20 +95,20 @@ export class AttributorSerializer implements IAttributorSerializer {
 			seqs.length === timestamps.length && timestamps.length === attributionRefs.length,
 			0x4b1 /* serialized attribution columns should have the same length */,
 		);
-		const entries = new Array(seqs.length);
+		const entries = Array.from<[number, AttributionInfo]>({ length: seqs.length });
```

Comment:
I don't remember what the eslint rule doesn't like about the previous version, but would something like this be clearer (if possible)?

```typescript
const entries: AttributionInfo[] = Array.from({ length: seqs.length });
```

Reply:
Updated 🙂